app/assets/stylesheets/homepage.css.scss

```diff
@@ -150,17 +150,14 @@ list-style-type: none;
     font-weight: 700;
 }
 
-.us-and-reviews {
-    background: transparent url('texture.png') center;
-}
 
 .home-mission {
-    text-agn: center;
+    text-align: center;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered height, text-align